src/function.ts

```diff
@@ -709,12 +709,12 @@ export class Function<
               `While serializing ${_resource.node.path}:\n\n${e.message}`
             );
           } else if (e instanceof Error) {
-            throw Error(
-              `While serializing ${_resource.node.path}:\n\n${e.message}`
-            );
+            throw e;
+            // throw Error(
+            //   `While serializing ${_resource.node.path}:\n\n${e.message}`
+            // );
```

Comment:
delete?

Reply:
I added this because it was very difficult to tell which function failed when the failure was async and there are many functions...

Reply:
This was a mistake, i was debugging